test/unittests/test_xmlupload.py

```diff
@@ -148,8 +148,7 @@ def test_remove_circular_references(self) -> None:
         stashed_xml_texts_expected = {
             "test_thing_1": {
                 "testonto:hasRichtext": [
-                    "\n                This is <em>bold and <strong>strong</strong></em> text! "
-                    "It contains links to all resources:\n"
+                    "\n                This text contains links to all resources:\n"
```

Comment:
Little confused by "should use a text value that contains only some links" because the first line (new 151) is plain text.

Reply:
ah okay, I should have omitted the word "only" :-)